genext2fs.c

```diff
@@ -166,8 +166,8 @@ static int blocksize = 1024;
 #define BLOCKS_PER_GROUP  8192
 #define INODES_PER_GROUP  8192
 /* Percentage of blocks that are reserved.*/
-#define RESERVED_BLOCKS       5/100
-#define MAX_RESERVED_BLOCKS  25/100
+#define RESERVED_BLOCKS       (5/100.)
```

Comment:
I'm not fan of introducing floats just for this.  
Could you please instead use something like
```c
#define TO_RESERVED_BLOCKS(n) ((n) * 5 / 100)
```
?